src/commands/clone.ts

```diff
@@ -37,10 +21,13 @@ export default async (scriptId: string, versionNumber: number, cmd: { rootDir: s
   scriptId = scriptId ? extractScriptId(scriptId) : await getScriptId();
   spinner.setSpinnerTitle(LOG.CLONING);
   const rootDir = cmd.rootDir;
-  saveProject({
-    scriptId,
-    rootDir,
-  }, false);
+  saveProject(
+    {
+      scriptId,
+      rootDir,
+    },
+    false,
```

Comment:
I think removing the trailing comma here will make this wrap on one line.

Reply:
`npm run prettier` (at least as it is configured now) format it like this